examples/cis3-nft-sponsored-txs/src/lib.rs

```diff
@@ -218,12 +224,15 @@ impl<S: HasStateApi> AddressState<S> {
 #[concordium(state_parameter = "S")]
 struct State<S> {
     /// The state for each address.
-    state:               StateMap<Address, AddressState<S>, S>,
+    state: StateMap<Address, AddressState<S>, S>,
     /// All of the token IDs
-    all_tokens:          StateSet<ContractTokenId, S>,
+    all_tokens: StateSet<ContractTokenId, S>,
     /// Map with contract addresses providing implementations of additional
     /// standards.
-    implementors:        StateMap<StandardIdentifierOwned, Vec<ContractAddress>, S>,
+    implementors_standards: StateMap<StandardIdentifierOwned, Vec<ContractAddress>, S>,
+    /// Map with contract addresses providing implementations of additional
+    /// entrypoints of the permit function.
+    implementors_permit_entrypoints: StateMap<StandardIdentifierOwned, Vec<ContractAddress>, S>,
```

Comment:
This is a bit weird to me. So other contracts can provide the permit version of a given entrypoint?
If so, it seems a bit redundant with CIS-0, since it can already provide contracts which provide CIS-3 for this contract.
I expected it to just have a StateSet with entrypoints supported by `permit`.